react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts

```diff
@@ -0,0 +1,122 @@
+import { Constants } from './../../cypress/fixtures/constants';
+import { baseSelectors } from './../../cypress/common/selectors';
+import { BaseMethods } from "../../cypress/common/base";
+
+const basePage: BaseMethods = new BaseMethods()
+
+const appsData = [
+    {
+        appName: Constants.elementsText.reactNestedRoutersApp1,
+        page1: Constants.elementsText.reactNestedRoutersPage1App1,
+        page2: Constants.elementsText.reactNestedRoutersPage2App1,
+        host: 8081
+    },
+    {
+        appName: Constants.elementsText.reactNestedRoutersApp2,
+        page1: Constants.elementsText.reactNestedRoutersPage1App2,
+        page2: Constants.elementsText.reactNestedRoutersPage2App2,
+        host: 8082
+    }
+]
+
+appsData.forEach((
+    property: {
+        appName: string,
+        page1: string,
+        page2: string,
+        host: number
+    }
+) => {
+    describe(`Check ${property.appName}`, () => {
+        beforeEach(() => {
+            basePage.openLocalhost(property.host)
+        })
+    
+        it(`Check ${property.appName} build and running (Check elements on the page)`, () => {
```

Comment:
please specify in name which element on page checked

Reply:
Fixed